### ISSUE TITLE
Velero CLI: expected pointer, but got nil when getting repos

### ISSUE THREAD

**Issue by waza-ari** [vmware-tanzu/velero#7514]
**What steps did you take and what happened:**

When trying to use the client to fetch repositories, the following error occurs:

```
$ velero repo get                  
An error occurred: expected pointer, but got nil
```

**What did you expect to happen:**

Expect to see a list of repositories:

```
$ k get backuprepository       
NAME                                    AGE   REPOSITORY TYPE
database-mkspazure-kopia-qr5gm          31h   kopia
hashicorp-vault-mkspazure-kopia-9xvww   31h   kopia
homepage-dev-mkspazure-kopia-jpccl      31h   kopia
```

**Environment:**

- Velero version (use `velero version`): 

```
Client:
	Version: v1.13.0
	Git commit: -
Server:
	Version: v1.13.0
```

- Velero features (use `velero client config get features`): 

```
features: <NOT SET>
```

- Kubernetes version (use `kubectl version`):

```
Client Version: v1.29.2
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.27.11
WARNING: version difference between client (1.29) and server (1.27) exceeds the supported minor version skew of +/-1
```

- Kubernetes installer & version:

```
rke version v1.5.6
```

- Cloud provider or hardware configuration: on-prem, virtualised
- OS (e.g. from `/etc/os-release`): macOS Sonoma

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

**Comment by danfengliu** [comment]
Thank you for reporting this bug, we will be working on it!

**Comment by blackpiglet** [comment]
@waza-ari 
https://github.com/vmware-tanzu/velero/pull/7407
Please try with the just released v1.13.1.

**Comment by waza-ari** [comment]
@blackpiglet thank you very much, that was quick! Tested it with the new version and it works perfectly now. Thanks for the fix 🎉
